Azure/sonic-buildimage
# Convert docker-iccpd to buster

Signed-off-by: Joyas Joseph <joyas_joseph@dell.com>

**- Why I did it**
This is an enhancement.

**- How I did it**
Remove docker-iccpd from the list of stretch dockers.

**- How to verify it**
Build `target/docker-iccpd.gz`
Verify that `/etc/apt/sources.list` in `iccpd` docker points to `buster` using ` docker exec iccpd cat /etc/apt/sources.list`

**- Description for the changelog**
Convert docker-iccpd to buster


**- A picture of a cute animal (not mandatory but encouraged)**
